components/Form/index.js

```diff
@@ -37,6 +34,14 @@ export default function Form() {
           </option>
           <option value="all two days">all two days</option>
         </select>
+        <label htmlFor="Place"> Place: </label>
+        <select name="place" id="Place">
+          <option value="living room" id="Place">
+            living room
+          </option>
```

Comment:
Maybe you could add 2-3 rooms more, to show, that it works. Should work as before with these options. 
```
 <option value="living room" id="Place">living room</option> 
<option value="kitchen" id="Place"> Kitchen  </option> 
<option value="basement" id="Place">Basement</option>
```
Edit: Maybe you have to do the same changes on your card